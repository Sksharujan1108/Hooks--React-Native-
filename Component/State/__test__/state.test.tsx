import React from 'react';
import renderer from 'react-test-renderer';
import States from '..';




test('renders correctly', () => {
  const tree = renderer.create(<States/>).toJSON();
  expect(tree).toMatchSnapshot();
});