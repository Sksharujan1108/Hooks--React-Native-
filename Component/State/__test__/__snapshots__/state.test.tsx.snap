// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "pink",
      "flex": 1,
    }
  }
>
  <Text
    style={
      {
        "backgroundColor": "yellow",
        "fontSize": 25,
        "fontWeight": "bold",
        "marginTop": 80,
      }
    }
  >
    useState
  </Text>
  <Text
    style={
      {
        "fontSize": 25,
        "marginRight": 150,
        "marginTop": 20,
      }
    }
  >
    Name : 
    kamal
  </Text>
  <Text
    style={
      {
        "fontSize": 25,
        "marginRight": 150,
        "marginTop": 20,
      }
    }
  >
    Age : 
    20
  </Text>
  <Text
    style={
      {
        "backgroundColor": "white",
        "fontSize": 25,
        "marginRight": 100,
        "marginTop": 20,
      }
    }
  >
     I am 
    vimal
     age is 
    25
  </Text>
  <Text
    style={
      {
        "backgroundColor": "white",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginTop": 35,
        "paddingHorizontal": 30,
        "paddingVertical": 10,
      }
    }
  >
     
    0
     
  </Text>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "blue",
        "borderRadius": 15,
        "elevation": 20,
        "flexDirection": "row",
        "marginRight": 150,
        "marginTop": 40,
        "marginVertical": 20,
        "opacity": 1,
        "paddingHorizontal": 10,
      }
    }
  >
    <Text
      style={
        {
          "color": "red",
          "flexDirection": "row",
          "fontSize": 20,
          "padding": 10,
        }
      }
    >
       Increase 
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "blue",
        "borderRadius": 15,
        "elevation": 20,
        "flexDirection": "row",
        "marginRight": 150,
        "marginTop": 40,
        "marginVertical": 20,
        "opacity": 1,
        "paddingHorizontal": 10,
      }
    }
  >
    <Text
      style={
        {
          "color": "red",
          "flexDirection": "row",
          "fontSize": 20,
          "padding": 10,
        }
      }
    >
       Decrease 
    </Text>
  </View>
</View>
`;
